app/assets/scripts/components/project-card.js

```diff
@@ -63,7 +74,8 @@ var ProjectCard = React.createClass({
     const { project, lang } = this.props;
     const locationLang = this.props.lang === 'en' ? 'name' : 'nameAr';
     const ontime = isOntime(project);
-    const statusClass = ontime ? 'project--ontime' : 'project--delayed';
+    const ontimeLookup = {extended: 'Extended', delayed: 'Delayed', ontime: 'On Time'};
```

Comment:
Let's make this a helper function in utils/format` so that all translation functions are in one place. This way we don't have to look into every file if we need to add an arabic translation, for example.